staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -400,6 +400,33 @@ type ListOptions struct {
 	// This field is not supported when watch is true. Clients may start a watch from the last
 	// resourceVersion value returned by the server and not miss any modifications.
 	Continue string `json:"continue,omitempty" protobuf:"bytes,8,opt,name=continue"`
+
+	// SendInitialEvents, when set together with Watch option,
+	// begin the watch stream with synthetic init events to build the
+	// whole state of all resources followed by a synthetic "Bookmark"
+	// event containing a ResourceVersion after which the server
+	// continues streaming events.
+	//
+	// When SendInitialEvents option is set, we require ResourceVersionMatch
+	// option to also be set. The semantic of the watch request is as following:
+	// - ResourceVersionMatch = NotOlderThan
+	//   It starts with sending initial events for all objects (at some resource
+	//   version), potentially followed by an event stream until the state
+	//   becomes synced to a resource version as fresh as the one provided by
+	//   the ResourceVersion option. At this point, a synthetic bookmark event
+	//   is send and watch stream is continued to be send.
```

Comment:
Can you just describe the differences from the scenario in the beginning part of this comment?

Reply:
Also please use ``` `resourceVersion` ``` when referring to the field.

Reply:
I could change it to:
```
is interpreted as "data at least as new as the provided resourceVersion" 
and the bookmark event is send when the state is synced 
to a resourceVersion as fresh as the one provided by the ListOptions
```

Reply:
Let's change and Daniel will take another look later on it :)

[I'm fine with this, but I'm too close to it to be fully confident it's really understandable.]